developer_manual/scripts/theme-bootstrap.sh

```diff
@@ -72,11 +43,22 @@ function clone_example_theme
   fi
 }
 
-echo unzip wget | xargs bash -c 'install_package "$@"'
+E_BADARGS=85
+
+# Stores the directory where this script was called from
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+if (( $# != 2 ))
+then
+  echo "Not enough arguments supplied."
+  echo "Usage: $( basename "$0" ) [-v] [-h] [theme id] [owncloud directory]"
```

Comment:
``owncloud directory]`` --> ``[/full path to owncloud_root/]`` to be inline with the php script above
``[theme id]`` --> ``[app name]`` there is no app id known, only the full app name